lib/streamlit/elements/json.py

```diff
@@ -39,26 +37,11 @@
 Body: TypeAlias = Union[List[Any], Tuple[Any, ...], MutableMapping[Hashable, Any]]
```

Comment:
This `TypeAlias` might not be needed any more. This was used to declare the type of the `body` parameter on `_convert_sets_to_lists`.